internal/test/util/kong_config.go

```diff
@@ -69,6 +78,9 @@ func fetchKongConfig() (KongConfig, error) {
 	if err != nil {
 		return KongConfig{}, fmt.Errorf("fetch SNIs: %v", err)
 	}
+	vaults, _ := client.Vaults.ListAll(ctx)
```

Comment:
What you can do is checking if the error code and ignore 404s.

Reply:
Yeah, this [comment](https://github.com/Kong/koko/pull/464/files#r1003665778) talks about it too. I ran all the E2E tests and this call isn't expected to ever error, as it's just listing vaults.

What would you said about just doing the below? If so, don't mind making that change.
```suggestion
	vaults, err := client.Vaults.ListAll(ctx)
	if err != nil {
		return KongConfig{}, fmt.Errorf("unable to fetch vaults: %w", err)
	}
```

Reply:
This is actually expected to error out with 404s for DPs not supporting Vaults, so pre-3.0

Reply:
Derp, right... will get in the proper fix for this. Completely sidestepped my mind this was calling the Kong API directly.

Reply:
Derp, right... will get in the proper fix for this.